compat/mingw.c

```diff
@@ -3304,13 +3304,8 @@ const char *program_data_config(void)
 
 	if (!initialized) {
 		const char *env = mingw_getenv("PROGRAMDATA");
-		const char *extra = "";
-		if (!env) {
-			env = mingw_getenv("ALLUSERSPROFILE");
-			extra = "/Application Data";
-		}
```

Comment:
This needs to be in a separate commit, as we do not want to mingle different purposes in a single commit. One commit should remove the XP support, one should enhance the documentation.